pkg/controllers/leasegarbagecollection/suite_test.go

```diff
@@ -42,7 +42,7 @@ import (
 
 var ctx context.Context
 var env *test.Environment
-var garbageCollectionController controller.Controller
+var garbageCollectionController *leasegarbagecollection.Controller
```

Comment:
Update, let's use `ExpectObjectReconciled` in our own expectations right now. I'm not going to migrate us to operatorpkg expectations because this has some conflicts with our existing expectations and is a larger overhaul.